pkg/network/usm/compile_test.go

```diff
@@ -32,3 +35,41 @@ func TestHttpCompile(t *testing.T) {
 		_ = out.Close()
 	})
 }
+
+func TestUSMCorrectlyInstrumentedWithTrampoline(t *testing.T) {
+	ebpftest.TestBuildMode(t, ebpftest.RuntimeCompiled, "", func(t *testing.T) {
```

Comment:
is the feature only supported in runtime compilation? do we plan to add it to co-re?

Reply:
It is supported for all buildmodes.
Prebuilt and CO-RE are tested here: https://github.com/DataDog/datadog-agent/pull/23070/files#diff-e577568bbd8afdcdbd633c3c612d9d60e2a8a84e8370fbf08e5b51d598492dc1R58

Runtime is tested separately here to easily get access to the runtime compilation API.